common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -159,11 +159,8 @@ export interface IDocumentAttributes {
 // @public
 export interface IDocumentMessage {
     clientSequenceNumber: number;
-<<<<<<< HEAD
```

Comment:
@justus-camp - Your doc build issue might be an easy fix?  Looks like there are conflict markers from an unresolved merge.

Reply:
I think I finally figured it out. Reading the build more carefully I see:

```
Warning: You have changed the public API signature for this project. Please copy the file "_api-extractor-temp/protocol-definitions.api.md" to "api-report/protocol-definitions.api.md", or perform a local build (which does this automatically). See the Git repo documentation for more info.
```

Just followed those instructions and hopefully CI will pass now 🤞

Reply:
But you are correct that I messed up the merge conflict initially. VSCode doesn't save by default when you resolve conflicts in the editor and it gets me pretty often 😄